### ISSUE TITLE
Command Line Interface for Mac

### ISSUE THREAD

**Issue by jp6422** [mzmine/mzmine_documentation#42]
Is there an equivalent to the .exe file that can be used to run mzMine from the command line on windows? It looks like there is no corresponding documentation for running mzmine in the terminal.

**Comment by tomas-pluskal** [comment]
Hi, yes there is a way.

For example, if your MZmine app is located in /Applications, you can use this command in the terminal:
`/Applications/MZmine.app/Contents/MacOS/MZmine -batch /path/to/batch_file.xml`

Please contribute by adding this information to the MZmine documentation, thanks :)

**Comment by aaymeloglu** [comment]
What about launching MZmine with the file you generated from the above command loaded? Basically I want to script generating the `.mzmine` file (which your above command lets me do) and then opening it in the app, equivalent to if I chose "Project" -> "Open Project" in the ui. I was able to do this for MZmine2, but I'm struggling to port my script so my team can use the new version. Apologies if this is documented but I've been combing through it and haven't been able to find my answer. Thanks!
